docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters.md

```diff
@@ -126,15 +126,13 @@ When an RKE2 or K3s cluster is registered in Rancher, Rancher will recognize it.
 - The ability to configure the maximum number of nodes that will be upgraded concurrently
 - The ability to see a read-only version of the cluster's configuration arguments and environment variables used to launch each node in the cluster
 
-### Additional Features for Registered EKS, AKS and GKE Clusters
+### Additional Features for Registered EKS, AKS, and GKE Clusters
 
-Registering an Amazon EKS, Azure AKS or GKE cluster allows Rancher to treat it as though it were created in Rancher.
+When you register an Elastic Kubernetes Service (EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE) cluster, Rancher handles the cluster similarly to clusters created in Rancher. However, Rancher doesn't destroy registered clusters when you delete them through the Rancher UI.
```

Comment:
The acronyms can be kept here and the acronym expansions should instead be moved earlier in the doc where they're first used.